code/__HELPERS/icons.dm

```diff
@@ -141,7 +141,7 @@ proc/adjust_brightness(var/color, var/value)
 				return y_pixel - 1
 	return null
 
-/proc/check_icon(var/icon/I, var/wanted_state = null)
+/proc/has_icon(var/icon/I, var/wanted_state = null)
```

Comment:
But why?

Reply:
unatomiiiiiiiiiiiiic

Reply:
I-i can explain, check icon already exists in rune code: 

>/obj/effect/rune/proc/check_icon(var/mob/M = null)
    get_uristrune_cult(word1, word2, word3, M)

Reply:
That's even worse